docs/book/intro.md

```diff
@@ -7,9 +7,9 @@ scanned text that Optical Character Recognition (OCR) software has been unable t
 if a user correctly provides the first word, the second is likely correctly entered as well, and can be used to
 improve OCR software for digitizing books.
 
-> ### Additional requirements
-> 
+> ## Additional requirements
+>
```

Comment:
For the info blocks the order of the headline is correct. Therefore please use:

```suggestion
<!-- markdownlint-disable-next-line header-increment -->
> ### Additional Requirements
>
```

Reply:
Resolved via #14